samples/EcsPipeline/CMakeLists.txt

```diff
@@ -9,11 +9,25 @@ target_link_libraries(EcsPipeline
   Engine
   )
 
-copy_build_products(EcsPipeline
-  DEPENDENCY Resources
-  TARGET_LOCATION $<TARGET_FILE_DIR:EcsPipeline>/Resources
+if(NOVELRT_INCLUDE_CONAN_PATHS AND (WIN32 OR APPLE))
```

Comment:
We can use `CONDITION ...` here rather than this weird if/else stuff if you fix #400